Documentation/contributing/docs/docsframework.rst

```diff
@@ -0,0 +1,382 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _docs_framework:
+
+***********************
+Documentation framework
+***********************
+
+This page contains notes on the framework in use for Cilium documentation. Its
+objective is to help contributors understand the tools and build process for
+the documentation, and to help maintain it.
+
+Alas, this is the sort of document that can quickly get out of date. Please
+double-check the information with the codebase for accuracy when you need it,
+and update this page if you find discrepancies.
+
+Sphinx
+======
+
+Cilium relies on Sphinx to generate its documentation.
+
+Sphinx features
+---------------
+
+Here are some specific Sphinx features in use in Cilium's documentation:
+
+- `Tab groups`_
+
+- `OpenAPI`_ documentation generation
+
+- Mark-up languages: reStructuredText (rST) and Markdown (`MyST`_ flavor)
+
+- Substitutions, for example:
+
+  - ``|SCM_WEB|``
+  - ``|CHART_VERSION|``
+
+- Multiple versions (for all supported branches, plus two aliases: ``stable``
+  and ``latest``)
+
+.. _OpenAPI: https://github.com/sphinx-contrib/openapi
+.. _Tab groups: https://github.com/executablebooks/sphinx-tabs/
+.. _MyST: https://myst-parser.readthedocs.io
+
+Sphinx version
+--------------
+
+The version of Sphinx in use is defined in
+``Documentation/requirements-min/requirements.txt``. See the :ref:`section on
+requirements <docs_requirements>` for more details.
```

Comment:
Order nit
```suggestion
``Documentation/requirements-min/requirements.txt``. For more details, see the :ref:`section on
requirements <docs_requirements>`.
```

Reply:
I'll change. But can you elaborate, please?

Reply:
> can you elaborate, please?

Sure. For instructions, I'd like to consistently follow a format of "to accomplish `x`, do `y`". 

It's a consistency issue for how we tell readers to do things, both on other pages (a work in progress) and on this one. Your other instructions on this page follow that format very well, including an example I called out 👇🏻 as good practice.

The style guide for Cilium OSS doesn't currently specify a preference for the aforementioned format. Updating guidance on link formatting sounds like yet another good idea that's out of scope.
`/Documentation/contributing/docs/docsstyle.rst#links`